Arduino/final_wall_art_2.ino

```diff
@@ -0,0 +1,104 @@
+#include "SerialTransfer.h"
+#include <math.h>
+#include <AccelStepper.h>
+#define HALFSTEP 8
+
+#define motorPin1  8     // IN1 on the ULN2003 driver 1
+#define motorPin2  9     // IN2 on the ULN2003 driver 1
+#define motorPin3  10     // IN3 on the ULN2003 driver 1
+#define motorPin4  11     // IN4 on the ULN2003 driver 1
+
+#define motorPin5  3     // IN1 on the ULN2003 driver 2
+#define motorPin6  4     // IN2 on the ULN2003 driver 2
+#define motorPin7  5    // IN3 on the ULN2003 driver 2
+#define motorPin8  6    // IN4 on the ULN2003 driver 2
+int stepperSpeed = 1000; //speed of the stepper (steps per second)
+float steps1 = 0; // keep track of the step count for motor 1
+float steps2=0;
+float target1=0;
+float target2=0;
+boolean turn1 = true;
+float n=0;
+// Initialize with pin sequence IN1-IN3-IN2-IN4 for using the AccelStepper with 28BYJ-48

```

Comment:
Include line breaks and work on code aesthetics. Remove unnecessary and blank lines etc.